lib/dataElementListener.js

```diff
@@ -72,12 +79,18 @@ class DataElementImporter extends SHRDataElementParserListener {
     }
 
     // Process the version
-    const version = ctx.docHeader().version();
-    const major = parseInt(version.WHOLE_NUMBER()[0], 10);
-    const minor = parseInt(version.WHOLE_NUMBER()[1], 10);
-    this._currentGrammarVersion = new Version(major, minor);
+    let docHeader = ctx.docHeader();
+    if (docHeader.version()) {
+      const version = docHeader.version();
+      const major = parseInt(version.WHOLE_NUMBER()[0], 10);
+      const minor = parseInt(version.WHOLE_NUMBER()[1], 10);
+      this._currentGrammarVersion = new Version(major, minor);
 
-    logger.debug({shrId: ns, version: this._currentGrammarVersion.toString()}, 'Start importing data element namespace');
+      logger.debug({shrId: ns, version: this._currentGrammarVersion.toString()}, 'Start importing data element namespace');
+    }
+    else {
+      logger.error('Grammar not defined here. ERROR_CODE 11???');
```

Comment:
Can we change the error message to 'Grammar declaration not found. ERROR_CODE:11???'